charts/illumidesk/values.yaml

```diff
@@ -300,6 +300,18 @@ jupyterhub:
       type: ClusterIP
   # Single user server
   singleuser:
+    # Default CPU Limits and Gurantees
+    cpu:
+      # Guranteed CPU allocation
+      guarantee: 0.2
+      # Max CPU that can be allocated
+      limit: 0.2
+    # Default Memory Limits and Gurantees
+    memory:
+      # Guranteed Memory allocation
+      guarantee: 200Mi
+      # Max Memory that can be allocated
+      limit: 200Mi
```

Comment:
200 Mi is not enough, that amount would barely be enough just to run the Notebook server (Tornado web app). Let's set this to use the JupyterHub default, 1G, and remove the guarantee value so that it's upper bound to the limit.